Exception with numpy 1.20.0

Hello,

We're using Py6S for some time now and everything works smoothly, thanks a lot :+1:!

But today we noticed an issue when Py6S is used with numpy 1.20.0. The issue is easily reproducible thanks to the `RunAllWavelengthsTests` unit test of Py6S.

Unit tests are green in a virtualenv with numpy 1.19.5:

``` bash
$ virtualenv venv_numpy_1_19_5
$ source venv_numpy_1_19_5/bin/activate
(venv_numpy_1_19_5) $ pip install numpy==1.19.5 scipy matplotlib pytest
(venv_numpy_1_19_5) $ pytest test/test_run_all_wavelengths.py
========================================== test session starts ===========================================
platform linux -- Python 3.9.1, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
rootdir: /tmp/Py6S
collected 2 items

test/test_run_all_wavelengths.py ..                                                                [100%]

=========================================== 2 passed in 9.29s ===========================================
```

But there's an exception when the last version of numpy (1.20.0) is used:

``` bash
$ virtualenv venv_numpy_1_20_0
$ source venv_numpy_1_20_0/bin/activate
(venv_numpy_1_20_0) $ pip install numpy scipy matplotlib pytest
(venv_numpy_1_20_0) $ pytest test/test_run_all_wavelengths.py
========================================== test session starts ===========================================
platform linux -- Python 3.9.1, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
rootdir: /tmp/Py6S
collected 2 items

E                   Py6S.sixs_exceptions.OutputParsingError: The specifed output variable does not exist.
...
...
```

--- a/Py6S/outputs.py
+++ b/Py6S/outputs.py
@@ -82,6 +82,8 @@ class Outputs(object):
         """Executed when an attribute is referenced and not found. This method is overridden
         to allow the user to access the outputs as ``outputs.variable`` rather than using the dictionary
         explicity"""
+        if name == "__array_struct__":
+            raise AttributeError()
 
         # If there is a key with this name in the standard variables field then use it
         if name in self.values:

